src/containers/table-container-head/table-container-head.js

```diff
@@ -5,7 +5,9 @@ import { TableHead, TableRow, TableCell } from '@material-ui/core';
 
 const TableContainerHead = ({ titles }) => {
   const headRow = titles.map((title, index) => (
-    <TableCell key={index}>{title}</TableCell>
+    <TableCell data-cy={title} key={index}>
```

Comment:
Don't use `index` as a key, use `title` instead if it is always unique and stable or find some another value for the index, please

Reply:
fixed